### PR TITLE
docs: document variables used by the cookiecutter template #6

### PR DESCRIPTION
**Pull request by martinm82**
This PR is documenting all the available options when creating the template

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,11 +42,41 @@ cookiecutter gh:tomtom-international/cookiecutter-python
...
+
+Cookiecutter will prompt you for various settings which we are explained here:
+
+* `full_name` -  The authors name (default: John Doe)
+* `email` - The authors email (default: john.doe@example.com)
+* `vs` - Select between github (1) or bitbucket (2) (default: 1)
+* `vs_url` - Provide the domain name of the versioning system without `http(s)://` (default: github.com)
+* `vs_account` - Account/organization on Github or project name on BitBucket Server (default: tomtom-international)
+* `project_name` - The name of your Python project (default: python-skeleton)
+* `project_slug` -  Python friendly module name. By default this is the normalized version of `project_name` (default: python_skeleton)
```
[the remark this answers is not in the document]
Comment (martinm82) [reply README.md]: The project_name is the repository name. Added that to the description

File: README.md [hunk 2]
```diff
@@ -49,6 +59,57 @@ Then:
 * If you chose to use Jenkins it depends on your internal Jenkins setup.
   * Required Jenkins plugins: `Warnings Next Generation Plugin >=4.0.0`, `Cobertura Plugin >=1.13`,  `Pipeline: GitHub Groovy Libraries >=1.0`
   * For internal TomTom projects: Trigger **Scan Organization Folder Now** on the BitBucket Team Project folder. In case of problems contact the @nav-pipeline team.
+* Contact the @nav-pipeline team to activate your project on `pyup.io`.
```
Comment (maikelvdh) [inline README.md]: Do you still want to mention it giving the recent addition of Dependabot?

File: README.md [hunk 3]
```diff
@@ -49,6 +59,57 @@ Then:
...
+* `docker_registry_credentials_id` - For Azure Pipelines this is the service connection that allows you to push Docker images to a public Docker registry. For Jenkins this is the credentials ID used for authentication with for example an Artifactory Docker repository.
+* `ci_ssh_agent_user` -  Skip this for Azure Pipelines. For Jenkins this is the credentials ID for a user authenticated with a SSH private key. This is required to allow git commits during the versioning process.
+* `azure_build_definition_id` - Skip this for Jenkins. For Azure Pipelines this is the ID identifiying the pipeline (eg. https://dev.azure.com/tomtomweb/GitHub-TomTom-International/_build?definitionId=26)
+
+## Tips
+
+In case you want to host a project in the `tomtom-international` organization and you chose Azure for the CI contact the @nav-pipeline team to request a build pipeline in the [TomTom Azure DevOps account](https://dev.azure.com/tomtomweb/GitHub-TomTom-International/_build) (@nav-pipeline: detailed instructions on how to set up a project can be found [here](https://github.com/tomtom-international/azure-pipeline-templates/blob/master/README.md)).
```
Comment (maikelvdh) [inline README.md]: This is the same as before variable explanation. Is it so relevant to repeat twice?

File: README.md [hunk 4]
```diff
@@ -49,6 +59,57 @@ Then:
...
+
+Contact as well the @nav-pipeline team to activate your project on `pyup.io` for automated scanning of your Python dependencies.
+
+In case of hosting on Jenkins/Bitbucket Server we recommend using the *BitBucket Branch Source Plugin* that allows you to configure a folder in Jenkins that will scan the whole BitBucket project for repositories with a `Jenkinsfile`.
```
Comment (maikelvdh) [inline README.md]: Maybe lift this to `Jenkins` section instead.

File: README.md [hunk 5]
```diff
@@ -49,6 +59,57 @@ Then:
...
+* `ci_ssh_agent_user` -  Skip this for Azure Pipelines. For Jenkins this is the credentials ID for a user authenticated with a SSH private key. This is required to allow git commits during the versioning process.
+* `azure_build_definition_id` - Skip this for Jenkins. For Azure Pipelines this is the ID identifiying the pipeline (eg. https://dev.azure.com/tomtomweb/GitHub-TomTom-International/_build?definitionId=26)
+
+## Tips
+
+In case you want to host a project in the `tomtom-international` organization and you chose Azure for the CI contact the @nav-pipeline team to request a build pipeline in the [TomTom Azure DevOps account](https://dev.azure.com/tomtomweb/GitHub-TomTom-International/_build) (@nav-pipeline: detailed instructions on how to set up a project can be found [here](https://github.com/tomtom-international/azure-pipeline-templates/blob/master/README.md)).
+
+Contact as well the @nav-pipeline team to activate your project on `pyup.io` for automated scanning of your Python dependencies.
```
Comment (maikelvdh) [inline README.md]: Same as earlier comment do we want to keep this still? If yes then also worth to group together with the other part.

File: README.md (hunk 2 above)
Comment (martinm82) [reply README.md]: Hm.. I actually removed it here: https://github.com/tomtom-international/cookiecutter-python/pull/42/files#diff-04c6e90faac2675aa89e2176d2eec7d8L40

Probably wrongly merged

Comment (martinm82) [reply README.md]: Oh, I missed that place. There were two locations

File: README.md (hunk 4 above)
Comment (martinm82) [reply README.md]: Did that already

File: README.md (hunk 3 above)
Comment (martinm82) [reply README.md]: Argh, I see that my merge got totally messed up

**Pull request merged**